lib/path.js

```diff
@@ -1140,7 +1123,10 @@ const win32 = {
   },
 
   matchesGlob(path, pattern) {
-    return glob(path, pattern, true);
+    emitExperimentalWarning('glob');
+    const test = lazyGlob()
+    process._rawDebug(test);
```

Comment:
It looks like the linter is unhappy and there is a leftover debugging statement.